packages/manager/apps/telecom/src/app/telecom/pack/move/offers/move-offers.controller.js

```diff
@@ -180,6 +190,52 @@ export default class PackMoveOffersCtrl {
   }
 
   selectOffer(offer) {
-    this.$scope.$emit('offerSelected', offer);
+    const selectedOffer = offer;
+
+    const params = {
+      eligibilityReference: selectedOffer.eligibilityReference,
+      offerName: selectedOffer.offerName,
+    };
+
+    // Retrieve option values (included and added)
+    let options = map(selectedOffer.options, (option) => {
+      if (
+        option.included > 0 ||
+        (option.choosedValue && option.choosedValue > 0)
+      ) {
+        let quantity = option.included;
+        if (option.choosedValue && option.choosedValue > 0) {
```

Comment:
Since the condition is repeated, it would be better if you move this logic to a function and check like `isOptionChosen()`